PythonVisualizations/SimpleSorting.py

```diff
@@ -167,19 +166,18 @@ def insert(self, item):
     
     def insert(self, val):
         self.startAnimations()
-        self.cleanUp()
-        self.showCode(self.insertCode.strip())
-        self.createCodeTags(self.insertCodeSnippets)
+        callEnviron = self.createCallEnvironment(
+            self.insertCode.strip(), self.insertCodeSnippets)

```

Comment:
doesn't `createCallEnvironment`  create a `CodeHighlightBlock` object which calls `code.strip()`? Is it necessary to call `strip()`  on `insertCode`?

Reply:
Yes that `.strip()` in the arguments to the call is redundant.  It won't hurt anything to leave it either.  Modifying the branch and updating the PR, however, takes a bunch of time.